packages/cli/src/rest/projects.ts

```diff
@@ -11,7 +11,8 @@ type ProjectResponse = Project & { id: string, created_at: string }
 
 export interface Change {
   logicalId: string,
-  physicalId?: string,
+  physicalId?: string|number,
+  checkId?: string,
```

Comment:
This is a mistake I guess?

Reply:
The `physicalId` is [now](https://github.com/checkly/checkly-backend/pull/4593/files#diff-d3940f49c21abfbcd392c2edc807229e39ff31e9227f2ca682bea0c26572599fR99) filled either with the `physicalId` if it's available, otherwise it's the resource ID (either checkId, dashboardId etc..), those can either be a string or a number, right?

Reply:
Or do you mean the `checkId`? Because that one is indeed a mistake 😆